### PR TITLE
Add BookBeat

### PR DESCRIPTION
**Pull request by slothtown**

### CODE REVIEW

**Comment by slothtown** [comment]
I'm still pretty unfamiliar with Github and PR's, so please let me know if there's something here I can do.

File: entries/b/bookbeat.fi.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "BookBeat": {
+    "domain": "bookbeat.fi",
+    "url": "https://www.bookbeat.fi/",
```
Comment (hkamran80) [inline entries/b/bookbeat.fi.json]:
```suggestion
```

The `url` attribute is only needed if going to the domain (e.g. `https://bookbeat.fi`) fails

File: entries/b/bookbeat.fi.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "contact": {
+      "email": "tuki@bookbeat.com",
+      "facebook": "bookbeatfi",
```
Comment (hkamran80) [inline entries/b/bookbeat.fi.json:7]:
```suggestion
      "facebook": "BookBeatFI",
```

File: img/b/bookbeat.fi.svg [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            </g>
+        </g>
+    </g>
+<style xmlns="" data-source="base" class="dblt-ykjmwcnxmi"/><style xmlns="" data-source="stylesheet-processor" class="dblt-ykjmwcnxmi"/></svg>
```
Comment (hkamran80) [inline img/b/bookbeat.fi.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32.6 38.2"><path fill="#0b0b0a" d="M14.2 36.95h-1.35V15.97c1.72 0 9.75.46 9.75 9.86 0 10.44-6.18 11.12-8.4 11.12zM12.83 1.66c.24 0 .78 0 1.03.03 2.8.22 5.68 1.97 5.68 6.13 0 4.81-3.68 6.63-6.7 6.63zm6.8 13.07c2.72-.08 7.34-1.8 7.34-7.02C26.98 2 20.53.07 13.76.07H.79a.79.79 0 0 0-.79.8v.01c0 .36.25.65.6.77 1.37.48 2 .02 2 2.03v31.04c0 2-.63 1.55-2 2.03-.35.12-.6.4-.6.77v.01c0 .44.35.8.79.8h14.67c11.51 0 17.14-3.72 17.14-12.5 0-7.9-6.32-11.1-12.96-11.1z"/></svg>
```

Here's a minified SVG

File: entries/b/bookbeat.fi.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "BookBeat": {
+    "domain": "bookbeat.fi",
+    "url": "https://www.bookbeat.fi/",
+    "contact": {
```
Comment (hkamran80) [inline entries/b/bookbeat.fi.json]:
```suggestion
    "additional-domains": [
      "bookbeat.at",
      "bookbeat.be",
      "bookbeat.fi",
      "bookbeat.dk",
      "bookbeat.fr",
      "bookbeat.de",
      "bookbeat.it",
      "bookbeat.nl",
      "bookbeat.no",
      "bookbeat.pl",
      "bookbeat.es",
      "bookbeat.se",
      "bookbeat.ch",
      "bookbeat.co.uk"
    ],
    "contact": {
```

BookBeat has regional sites as well

File: entries/b/bookbeat.fi.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+      "email": "tuki@bookbeat.com",
+      "facebook": "bookbeatfi",
+      "language": "fi"
+    },
+    "keywords": [
+      "entertainment"
+    ],
+    "regions": [
+      "fi"
```
Comment (hkamran80) [inline entries/b/bookbeat.fi.json]:
```suggestion
      "fi",
      "at",
      "be",
      "bg",
      "hr",
      "cy",
      "cz",
      "dk",
      "ee",
      "fr",
      "de",
      "gr",
      "hu",
      "ie",
      "it",
      "lv",
      "lt",
      "lu",
      "mt",
      "nl",
      "no",
      "pl",
      "pt",
      "ro",
      "sk",
      "si",
      "es",
      "se",
      "ch",
      "gb"
```

BookBeat has regional sites as well

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/b/bookbeat.fi.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
+{
+  "BookBeat": {
+    "domain": "bookbeat.fi",
```
Comment (Carlgo11) [inline entries/b/bookbeat.fi.json]: As BookBeat operates internationally, a generic TLD is preferred over a local one.
```suggestion
    "domain": "bookbeat.com",
```

File: entries/b/bookbeat.fi.json [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    "url": "https://www.bookbeat.fi/",
+    "contact": {
+      "email": "tuki@bookbeat.com",
+      "facebook": "bookbeatfi",
+      "language": "fi"
```
Comment (Carlgo11) [inline entries/b/bookbeat.fi.json]:
```suggestion
      "email": "help@bookbeat.com",
      "facebook": "BookBeatApp"
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by slothtown** [comment]
Sorry for the commit mess, I'm still not that used to Github workflow and made some mistakes

**Comment by Carlgo11** [comment]
I see that.
If you're using Git:
```sh
git reset --hard 7115fc6f84b50982a8be1797bc1ada715e3c6947
git push -f
```

If you're using the web browser, let me know and I'll try and reset the branch for you.

**Comment by slothtown** [comment]
Thank you, I just did that.

**Review by Carlgo11 — APPROVED** [review]
Thank you! 🙂
Your contribution will be visible in around 10 minutes. Sometimes it takes longer due to server and browser cache.

**Comment by slothtown** [comment]
No problem! And thanks for the help!

**Pull request merged**